debian/control

```diff
@@ -0,0 +1,17 @@
+Source: superboucle
+Maintainer: Julien Acroute <superboucle@nura.eu>
+Section: sound
+Priority: optional
+Standards-Version: 3.9.4
+Build-Depends: debhelper (>= 9), dh-python,
+ python3-all (>= 3.3), python3-setuptools, python3-soundfile (>= 0.10),
+ python3-cffi (>= 1.12), python3-numpy (>= 1.16), python3-pyqt5 (>= 5.11)
+
+Package: superboucle
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends},
+ python3-soundfile (>= 0.10), python3-cffi (>= 1.12), python3-numpy (>= 1.16),
+ python3-pyqt5 (>= 5.11), libsndfile1-dev, libjack-jackd2-dev
+Description: SuperBoucle is a loop based software fully controllable with any
```

Comment:
You need to start with a "short description" (such as "loop based software midi device controllable"), then line break, then a full description